cmd/restic/global.go

```diff
@@ -160,9 +162,14 @@ func restoreTerminal() {
 	}
 
 	AddCleanupHandler(func() error {
+		// only restore the terminal settings when reading a password
+		if !isReadingPassword {
```

Comment:
Can this be expanded to give a short explanation as to why we need to do this?

One should be able to read the code and understand why this is done, and it's not really clear now.